cron.go

```diff
@@ -195,7 +207,7 @@ func (c *Cron) run() {
 		}
 
 		// 'now' should be updated after newEntry and snapshot cases.
-		now = time.Now().Local()
+		now = time.Now().In(c.location)
```

Comment:
I see a remaining time.Now().Local() on line 199 -- seems like that should also get updated?